package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "vuejs-datepicker",
-  "version": "0.4.14",
+  "version": "0.4.15",
```

Comment:
Can you remove the version increment please.
I will do this as a git tag with `npm version patch`